Mathlib/Order/Filter/Germ.lean

```diff
@@ -257,6 +257,11 @@ theorem compTendsto'_coe (f : Germ l β) {lc : Filter γ} {g : γ → α} (hg :
     f.compTendsto' _ hg.germ_tendsto = f.compTendsto g hg :=
   rfl
 #align filter.germ.comp_tendsto'_coe Filter.Germ.compTendsto'_coe
+
+theorem Filter.Tendsto.congr_germ {f g : β → γ} {l : Filter α} {l' : Filter β} (h : f =ᶠ[l'] g)
+    {φ : α → β} (hφ : Tendsto φ l l') : (f ∘ φ : Germ l γ) = g ∘ φ :=
+  @Quotient.sound _ (l.germSetoid γ) _ _ (hφ h)
```

Comment:
The proof should use [Filter.EventuallyEq.germ_eq](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Filter/Germ.html#Filter.EventuallyEq.germ_eq) instead of `Quotient.sound`.

I'm also wondering wether this deserves a lemma, is `(h.comp_tendsto hφ).germ_eq` not short enough?

Reply:
Good question! Let me look at sphere-eversion...

Reply:
It is unused there. In other words: I don't have a strong opinion and am happy to delete this.